browser-test/script.map

```diff
@@ -3,7 +3,7 @@
   "file": "script.js",
   "sourceRoot": "..",
   "sources": [
-    "browser-test/script.coffee"
+    "browser-test\\script.coffee"
```

Comment:
Actually, it looks like script.map is not excluded via the .gitignore as is the case for the other tests.